src/main/java/com/ecore/roles/service/MembershipsService.java

```diff
@@ -11,4 +12,6 @@ public interface MembershipsService {
     Membership assignRoleToMembership(Membership membership) throws ResourceNotFoundException;
 
     List<Membership> getMemberships(UUID roleId);
+
+    List<Role> getRoles(UUID userId, UUID teamId);
```

Comment:
This method has been created to return the list of roles by userId and teamId raised by business rule.